Source/Public/New-DocusaurusHelp.ps1

```diff
@@ -167,18 +175,62 @@ function New-DocusaurusHelp() {
 
     GetCallerPreference -Cmdlet $PSCmdlet -SessionState $ExecutionContext.SessionState
 
-    # make sure the passed module is valid
-    if (Test-Path($Module)) {
-        Import-Module $Module -Force -Global
-        $Module = [System.IO.Path]::GetFileNameWithoutExtension($Module)
-    }
+    # Get the module's name fromn the supplied markdown files.
+    if ($PSCmdlet.ParameterSetName.Equals('PlatyPSMarkdownPath'))
+    {
+        # Get the module's name from the supplied markdown files.
+        $moduleName = Get-ChildItem -LiteralPath $PlatyPSMarkdownPath -Filter *.md |
+            Select-Object -First 10 |
+            ForEach-Object { [System.IO.File]::ReadAllLines($_.FullName) -match '^Module Name: ' -replace '^Module Name:\s+' } |
+            Select-Object -Unique
```

Comment:
I think we should check all files, but limit to first ten lines, e.g. `Get-Content -TotalCount 10`. Just to avoid reading massive files when we only need the front matter at the top.

Reply:
I feel so dumb now... of course this makes sense and is what you meant. I've amended that, and hopefully the last issue with the build CI/CD system being upset with me!